exercises/strings/longestWord/longestWord.js

```diff
@@ -9,11 +9,26 @@
 
 function longestWord(string) {
   // This is your job. :)
+  let words = []
+  let endIndex = 0
+  let startIndex = 0
+  for (i in string) {
+    if (string[i] === " " || string[i] === ".") {
+      endIndex = Number(i)
+      words.push(string.slice(startIndex,endIndex))
```

Comment:
Since you're already building up an array like this, you may as well use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split

Actually, now that I read this, there's no reason to keep a growing list of the words at all. You only need to keep track of the longest word as you go. Whenever you find a new word, check if it should become the new "longest word" or not.